### PR TITLE
Feature/investigate django notifications

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@Ntu2koTrevor please review

**Comment by g1039** [comment]
👍

**Comment by g1039** [comment]
@nathanbegbie Please review

File: molo/commenting/tests/test_views.py [hunk 1]
```diff
@@ -265,3 +268,36 @@ def test_view_replies_report(self):
         self.assertTrue('report' in crow.prettify())
         self.assertTrue(reply.comment in replyrow.prettify())
         self.assertFalse('report' in replyrow.prettify())
+
+
+@override_settings(ROOT_URLCONF='molo.commenting.tests.test_views')
+class ViewNotificationsRepliesOnCommentsTest(TestCase, MoloTestCaseMixin):
```
Comment (nathanbegbie) [inline molo/commenting/tests/test_views.py:275]: @Ntu2koTrevor These tests need to be separated out into `test_models.py` and `test_views.py`

`test_models.py`
Create a parent comment and a reply (not a notification) and then confirm that the notification has been created, with the correct attributes.

`test_views.py`
If you are testing views, you need to make a request for a page and then look and see if the stuff that you want is in the actual page. This usually involves using `client.request.get("/commenting/...")`

In this case we want a user to

1. Create the users, article, comment and reply in `setUp`
1. Login as a user
1. Request the main page and confirm that the notification is there
1. Get the link from the notification references and request it
1. Check that the comment appears
1. Make another reply
1. Repeat the process and check that there is an unread and read notification in the notifications page

**Comment by g1039** [comment]
@nathanbegbie Ready for review

**Comment by nathanbegbie** [comment]
@Ntu2koTrevor +1

File: molo/commenting/templates/comments/comment_done.html [hunk 2]
```diff
@@ -1,4 +1,3 @@
-{% extends "base.html" %}
```
Comment (nathanbegbie) [inline molo/commenting/templates/comments/comment_done.html]: This shouldn't be removed if it was already there. You just needed to remove the ones that were added while we were working on this

File: molo/commenting/templates/comments/comments.html [hunk 3]
```diff
@@ -1,4 +1,3 @@
-{% extends "base.html" %}
```
Comment (nathanbegbie) [inline molo/commenting/templates/comments/comments.html]: This shouldn't be removed. See above

**Comment by nathanbegbie** [comment]
@Ntu2koTrevor Made a small cahnge to the readme. +1 once tests pass

**Pull request merged**